UnRen.bat

```diff
@@ -109,7 +120,7 @@ echo     6) Force enable rollback (scroll wheel)
 echo.
 echo     9) All of the above
 echo.
-set /p option=.  Enter a number: 

```

Comment:
this space is intended, so input begins 1 space after the colon, eg:
```
Enter a number: 1
```

Reply:
Right. I do this also like this and was unintended. Happened because Atom editor has a "remove trailing whitespace" option. Mostly useful, but not always...
Fixed.